blueprints/glimmer-helper/files/src/ui/components/__name__.ts

```diff
@@ -0,0 +1,2 @@
+export default function(params) {
```

Comment:
and since components have a class name now, shouldn't helper functions have one too?